app/src/main/res/values-it/strings.xml

```diff
@@ -8,42 +8,42 @@
     <string name="title_explore">Esplora</string>
     <string name="title_settings">Impostazioni</string>
     <string name="title_now_playing">In ascolto ora</string>
-    <string name="title_error_report">Error report</string>
+    <string name="title_error_report">Segnala errore</string>
 
     <!-- Preferenze -->
     <string name="pref_appearance_title">Aspetto</string>
     <string name="pref_follow_system_accent_title">Segui tema di sistema</string>
     <string name="pref_theme_color_title">Colore del tema</string>
     <string name="pref_dark_theme_title">Tema scuro</string>
-    <string name="pref_default_open_tab_title">Default open tab</string>
+    <string name="pref_default_open_tab_title">Scheda principale predefinita</string>
     <string name="dark_theme_on">Attivato</string>
     <string name="dark_theme_off">Disattivato</string>
     <string name="dark_theme_follow_system">Segui sistema</string>
     <string name="pref_content_title">Contenuti</string>
     <string name="pref_content_language_title">Lingua predefinita dei contenuti</string>
     <string name="pref_default_content_country_title">Paese predefinito dei contenuti</string>
     <string name="pref_enable_proxy_title">Attiva proxy</string>
-    <string name="pref_proxy_type_title">Proxy type</string>
+    <string name="pref_proxy_type_title">Tipo di proxy</string>
     <string name="pref_proxy_url_title">URL proxy</string>
-    <string name="pref_player_audio_title">Player and audio</string>
-    <string name="pref_general_title">Gerenal</string>
-    <string name="pref_audio_quality_title">Audio quality</string>
-    <string name="audio_quality_auto">Auto</string>
-    <string name="audio_quality_high">High</string>
-    <string name="audio_quality_low">Low</string>
+    <string name="pref_player_audio_title">Riproduttore e audio</string>
+    <string name="pref_general_title">Gerenale</string>
+    <string name="pref_audio_quality_title">Qualità dell'audio</string>
```

Comment:
`'` should be escaped.

Reply:
You are right. I try to do it with my phone